README.md

```diff
@@ -1,5 +1,44 @@
 # brainglobe-stitch
 
+Stitching tiled 3D light-sheet data in napari
+
+----------------------------------
+
+A [napari] plugin for stitching tiled 3D acquisitions from a [mesoSPIM] light-sheet microscope.
+The plugin utilises [BigStitcher] to align the tiles and napari to visualise the stitched data.
+
+## Installation
+
+We strongly recommend to use a virtual environment manager (like `conda` or `venv`). The installation instructions below
```

Comment:
I would just recommend conda. It's what we recommend for other BG tools. This way we can reduce the landscape of things we need to support. Nothing stopping users picking a different venv tool though.